pkg/webui/locales/en.json

```diff
@@ -412,9 +412,9 @@
   "console.containers.application-payload-formatters.uplink.uplinkResetWarning": "You do not have sufficient rights to view the current uplink payload formatter. Only overwriting is allowed.",
   "console.containers.application-title-section.index.lastSeenAvailableTooltip": "The elapsed time since the network registered activity (sent uplinks, confirmed downlinks or (re)join requests) of the end device(s) in this application.",
   "console.containers.application-title-section.index.noActivityTooltip": "The network has not recently registered any activity (sent uplinks, confirmed downlinks or (re)join requests) of the end device(s) in this application.",
-  "console.containers.applications-form.index.applicationName": "Application name",
-  "console.containers.applications-form.index.appIdPlaceholder": "my-new-application",
-  "console.containers.applications-form.index.appNamePlaceholder": "My new application",
+  "console.containers.applications-form.index.applicationName": "Project name",
+  "console.containers.applications-form.index.appIdPlaceholder": "my-new-project",
+  "console.containers.applications-form.index.appNamePlaceholder": "My new p",
```

Comment:
p to project?

Reply:
You're looking at an older version it seems like